main.py

```diff
@@ -35,20 +35,18 @@
                 else:
                     print("That letter doesn't appear in the word.")
                     i += 1
-                used_letters.append(letter)
+                    used_letters.append(letter)
```

Comment:
Better push the letter in the `used_letters` array only when it is already verified to be a new one. Otherwise you may end up with tons of repeating letters and the size would grow.